js/server/modules/@arangodb/actions.js

```diff
@@ -1156,13 +1154,52 @@ function firstRouting (type, parts, routes, rawParts) {
 
 function routeRequest (req, res, routes) {
   if (routes === undefined) {
-    var dbname = arangodb.db._name();
+    let internalRoute = false;
+    // systemServiceMounts contains the internal mount points, i.e.
+    // /_admin/aardvark and /_api/foxx
+    for (let mount of FoxxManager.systemServiceMounts) {
```

Comment:
this loop is redundant because all internal apps live in the prefixes we explicitly check below